src/generate.rs

```diff
@@ -212,17 +216,19 @@ fn build_multisig_transfer_tx(
 }
 
 pub fn build_omnilock_unlockers(
-    keys: Vec<secp256k1::SecretKey>,
+    keys: Vec<PrivkeyWrapper>,
     config: OmniLockConfig,
     omni_lock_type_hash: H256,
 ) -> HashMap<ScriptId, Box<dyn ScriptUnlocker>> {
+    let keys: Vec<secp256k1::SecretKey> = keys.iter().map(|k| k.0).collect();
     let signer = if config.is_ethereum() {
         SecpCkbRawKeySigner::new_with_ethereum_secret_keys(keys)
     } else {
         SecpCkbRawKeySigner::new_with_secret_keys(keys)
     };
+    let omnilock_signer = Box::new(CommonSigner::new(vec![Box::new(signer)]));
```

Comment:
If only use `CommonSigner` here, I think this is not necessary.

Reply:
keep the design for future use.